storm-core/src/jvm/backtype/storm/messaging/netty/Client.java

```diff
@@ -59,20 +59,16 @@
  * - Connecting and reconnecting are performed asynchronously.
  *     - Note: The current implementation drops any messages that are being enqueued for sending if the connection to
  *       the remote destination is currently unavailable.
- * - A background flusher thread is run in the background.  It will, at fixed intervals, check for any pending messages
- *   (i.e. messages buffered in memory) and flush them to the remote destination iff background flushing is currently
- *   enabled.
  */
 public class Client extends ConnectionWithStatus implements IStatefulObject {
+    private static final long PENDING_MESSAGES_FLUSH_TIMEOUT_MS = 600000L;
```

Comment:
It is large, but it matches the code that was there before.  This is only used on a close, as a timeout when trying to be sure that we have sent all pending messages.  We should look at potentially lowering it, but I think that is for a different JIRA.